pkg/glusterfs/api/types.go

```diff
@@ -28,7 +28,7 @@ const (
 	EntryStateUnknown EntryState = ""
 	EntryStateOnline  EntryState = "online"
 	EntryStateOffline EntryState = "offline"
-	EntryStateFailed  EntryState = "failed"
+	EntryStateFailed  EntryState = "removed"
```

Comment:
@lpabon I realised now that EntryStateFailed is a api. Do you really want to change it? It will break backward compatibility.